install/stack.yaml

```diff
@@ -0,0 +1,13 @@
+resolver: lts-14.27 # Last 8.6.5 
```

Comment:
Maybe we could use only one `stack.yaml`, removing `shake.yaml`?

Reply:
I think it still wouldn't AFAIR there were inconsistencies with the Cabal dependency (not 100% sure though). But as long as there's a dependency on Shake, we can't do much...

Reply:
Mmm not sure what was the problem, but i've tried:
* add 
```yaml
flags:
  hls-install:
    run-from-stack: true
```
to `./install/stack.yaml`
* remove `./install/shake.yaml`
* replace in `./install.hs`,`--stack-yaml=install/shake.yaml` with `--stack-yaml=install/stack.yaml`

And both `stack install.hs <target>` and opening `./install/src/BuildSystem.hs` seemsto work fine.

Reply:
@Ailrun we could do the suggested change above in another pr if you prefer and merge this as-is

Reply:
I'm not in hurry since I currently don't work with files under `/install` directory, and somehow managed HLS to work with itself, so please do whatever you are in favor of.

Reply:
@Ailrun sorry, bad mention 😅